src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/DiagnosticDescriptors.cs

```diff
@@ -208,5 +208,13 @@ public static class DiagnosticDescriptors
             category: "LoggingGenerator",
             defaultSeverity: DiagnosticSeverity.Error,
             isEnabledByDefault: true);
+
+        public static DiagnosticDescriptor PrimaryConstructorParameterLoggerHidden { get; } = DiagnosticDescriptorHelper.Create(
```

Comment:
Please add the entry to the file [https://github.com/dotnet/runtime/blob/0588f245dba6418349c22894b40337afc34f1cfc/docs/project/list-of-diagnostics.md?plain=1#L148](https://github.com/dotnet/runtime/blob/0588f245dba6418349c22894b40337afc34f1cfc/docs/project/list-of-diagnostics.md?plain=1#L148) too.

Reply:
and please tracking updating the doc [https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/source-generator-overview#reference](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/source-generator-overview#reference) too.

Reply:
I created https://github.com/dotnet/docs/issues/41084 to track the necessary documentation updates.